### PR TITLE
add dashboard content

### PR DESCRIPTION
**Pull request by osswald**

### CODE REVIEW

File: train_management/tests/helpers.py [hunk 1]
```diff
@@ -15,6 +16,20 @@ def __init__(self):
         self.C6109 = None
         self.WR109 = None
         self.F202 = None
+        self.Bw_Neuthal = None
```
Comment (busykoala) [inline train_management/tests/helpers.py:19]: attributes are not pep8 conform (snake case)... don't know if we care.

File: train_management/tests/helpers.py [hunk 2]
```diff
@@ -234,3 +249,116 @@ def create_train_components(self):
             max_speed=40,
         )
         self.F202.save()
+
+    def create_function(self):
+        self.Bw_Neuthal = Function(
+            label="Barrierenwärter",
```
Comment (busykoala) [inline train_management/tests/helpers.py:255]: Is English only the goal here?

File: train_management/tests/helpers.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 from train_management.models import Train
 from train_management.models import Vehicle
 from train_management.models import TrainConfiguration
+from train_management.models import Function
```
Comment (busykoala) [inline train_management/tests/helpers.py:5]: I think `Function` is a confusing naming...

Comment (osswald) [reply train_management/tests/helpers.py:5]: renamed in next pr

**Pull request merged**